pulpcore/pulpcore/etc/pulp/server.yaml

```diff
@@ -93,6 +93,12 @@
 #                 streaming the content to Apache.  Requires: mod_xsendfile to be installed.
 #                 When set to 'nginx', the X-Accel-Redirect header is injected which delegates
 #                 streaming the content to NGINX.
+#   `host`: The hostname and port number for the content app. This value should be specified
+#           only if the Content App is served on a host that's different from the REST API. The
+#           hostname portion needs to match at least one hostname in the `allowed_hosts` config.
+#           The default value is None. When a value is specified, Distributions use it to form
+#           the 'base_url'.
```

Comment:
My concern about mentioning `Distributions use it to form the base_url` is that if it's used for something else in the future it would be easy to miss updating this.  Also, I don't think it's useful information.